drivers/espi/Kconfig.it8xxx2

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config ESPI_IT8XXX2
+	bool "ITE IT8XXX2 embedded controller ESPI driver"
+	depends on SOC_IT8XXX2
+	help
+	  Enable ITE IT8XXX2 ESPI driver.
+
+if ESPI_IT8XXX2
+
+config ESPI_OOB_CHANNEL
```

Comment:
default is n, this is not explicitly required

Reply:
Done. (To remove the code)